Documentation/commands.md

```diff
@@ -209,8 +209,37 @@ FOO=bar
 EXAMPLE_OVERRIDE=over
 ```
 
+#### Logging
+
+To get the logs you need to get pod's machine name. You can use machinectl
```

Comment:
Yes. I wrote this in such a rush I'm surprised it makes any sense at all :stuck_out_tongue_winking_eye: